packages/compass-e2e-tests/tests/oidc.test.ts

```diff
@@ -109,17 +109,22 @@ describe('OIDC integration', function () {
         authNamePrefix: 'dev',
       };
 
-      cluster = await startTestServer({
-        args: [
-          '--setParameter',
-          'authenticationMechanisms=SCRAM-SHA-256,MONGODB-OIDC',
-          // enableTestCommands allows using http:// issuers such as http://localhost
-          '--setParameter',
-          'enableTestCommands=true',
-          '--setParameter',
-          `oidcIdentityProviders=${JSON.stringify([serverOidcConfig])}`,
-        ],
-      });
+      const args = [
+        '--setParameter',
+        'authenticationMechanisms=SCRAM-SHA-256,MONGODB-OIDC',
+        // enableTestCommands allows using http:// issuers such as http://localhost
+        '--setParameter',
+        'enableTestCommands=true',
+        '--setParameter',
+        `oidcIdentityProviders=${JSON.stringify([serverOidcConfig])}`,
+      ];
+
+      if (serverSatisfies('>= 8.0.0-alpha0', true)) {
```

Comment:
I think this condition just needs to be adjusted a bit for the fact that this isn't on 8.0 yet, only 8.1?